plugin/parser/differ/mysql/differ.go

```diff
@@ -79,7 +88,26 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 					})
 				}
 			}
-
+			for _, constraint := range newStmt.Constraints {
+				if constraint.Tp == ast.ConstraintIndex || constraint.Tp == ast.ConstraintKey ||
+					constraint.Tp == ast.ConstraintUniq || constraint.Tp == ast.ConstraintUniqKey ||
+					constraint.Tp == ast.ConstraintUniqIndex || constraint.Tp == ast.ConstraintFulltext {
+					indexName := getIndexName(constraint)
+					if oldConstraint, ok := indexMap[indexName]; ok {
+						if isIndexEqual(constraint, oldConstraint) {
+							delete(indexMap, indexName)
+							continue
+						}
+					}
+					createIndexStmts = append(createIndexStmts, &ast.CreateIndexStmt{
```

Comment:
How about using the `ALTER TABLE ADD CONSTRAINT` statement instead of the `CREATE INDEX` statement?

The reason is we also need to create the primary key and the foreign key later, we can use `ALTER TABLE ADD CONSTRAINT` statements uniformly.

On the other hand, we can merge multi-ALTER statements into one ALTER statement. It's more readable.

Reply:
Move to `ALTER TABLE ADD CONSTRAINT` statements done. I will merge the multi-`ALTER` statements into one ALTER statement in the following PR.

Reply:
PTAL.